web/docs/_TypescriptServerNote.md

```diff
@@ -0,0 +1,11 @@
+:::caution LSP Problems
+
+If you are using TypeScript, your editor may sometimes report type and import errors even while `wasp start` is running.
+
+This happens when the TypeScript Language Server gets out of sync with the current code.
+If you're using VS Code, you can manually restart the language server by opening the command palette and selecting _"TypeScript: Restart TS Server."_
+Open the command pallete with:
```

Comment:
If you want this on a new line, add a new line above. It renders like this now:

<img width="865" alt="Screenshot 2024-07-09 at 14 26 05" src="https://github.com/wasp-lang/wasp/assets/2223680/e832d190-388d-4d9e-861d-8f6729676617">

Reply:
No no, this is what I wanted.

I add a line break after each sentence because it doesn't change how the text is displayed and makes it easier to edit (moving around with arrow or vim keys, cutting out lines, selecting lines, etc.)